src/serialization/KiotaSerializer.Serialization.cs

```diff
@@ -70,7 +70,7 @@ public static partial class KiotaSerializer
     private static string GetStringFromStream(Stream stream)
     {
         using var reader = new StreamReader(stream);
-        return reader.ReadToEndAsync().ConfigureAwait(false).GetAwaiter().GetResult(); // so the asp.net projects don't get an error
```

Comment:
please revert as this new implementation will cause errors for clients used in an asp.net mvc application which doesn't allow reading network streams synchronously (default)